CriticalMass/MapViewController.swift

```diff
@@ -15,33 +15,13 @@ class MapViewController: UIViewController {
     private let nextRideManager: NextRideManager
 
     private let mapInfoViewController = MapInfoViewController.fromNib()
-
```

Comment:
this is just moving a couple lines of code

Reply:
👍